src/com/palmergames/bukkit/towny/TownyFormatter.java

```diff
@@ -385,9 +385,11 @@ private static void getRanks(Town town, List<String> ranklist) {
 				TownySettings.getLangString("mobspawns") + ((town.hasMobs() || world.isForceTownMobs()) ? TownySettings.getLangString("status_on"): TownySettings.getLangString("status_off")));
 
 		// | Bank: 534 coins
-		String bankString = "";
-		if (TownySettings.isUsingEconomy()) {
-			if (TownyEconomyHandler.isActive()) {
+		if (TownySettings.isUsingEconomy() && TownyEconomyHandler.isActive()) {
+			String bankString = "";
+			if(TownySettings.isTownBankruptsyEnabled() && town.isBankrupt()) {

```

Comment:
I feel like any time we're testing TownySettings.isTownBankruptcyEnabled() && town.isBrankrupt() we should only be testing town.isBankrupt() and within that method we can always return false if TownySettings.isTownBankruptcyEnabled() is not enabled. This will make things easier to read.